libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -632,8 +639,8 @@ class MapboxNavigation(
      * To start listening EHorizon updates [EHorizonObserver] should be registered.
      * Observer will be called when the EHorizon changes.
      * To save resources and be more efficient callbacks return minimum data.
-     * To get [EHorizonEdge] shape or [EHorizonEdgeMetadata] use [EHorizonGraphAccessor]
-     * To get more data about EHorizon road object use [EHorizonObjectsStore]
+     * To get [EHorizonEdge] shape or [EHorizonEdgeMetadata] use [GraphAccessor]
+     * To get more data about EHorizon road object use [RoadObjectsStore]
```

Comment:
Developers will not be able to create the instances themselves, we need to point them to `[MapboxNavigation.graphAccessor]` and `[MapboxNavigation.roadObjectsStore]` in all of the docs.

Reply:
Or leave as is but also add an entry in the class docs for `GraphAccessor` and `RoadObjectsStore` that the instance can be obtained via `MapboxNavigation`.

Reply:
done